cpp/src_prims/sparse/coo.cuh

```diff
@@ -834,9 +834,8 @@ __global__ static void symmetric_find_size(const math_t *restrict data,
                                            const int n, const int k,
                                            int *restrict row_sizes,
                                            int *restrict row_sizes2) {
-  const int j =
-    (blockIdx.x * blockDim.x) + threadIdx.x;  // for every item in row
-  const int row = (blockIdx.y * blockDim.y) + threadIdx.y;  // for every row
+  const int row = blockIdx.x * blockDim.x + threadIdx.x;  // for every row
```

Comment:
It would be helpful to have a simple pytest reproducer in `tests/cuml/manifold/test_tsne.py`. We can add the `stress` marker to it since it'll be pretty large.